6.0/fpm/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM php:5.6-fpm
+
+RUN apt-get update && apt-get install -y \
+  bzip2 \
+  libcurl4-openssl-dev \
+  libfreetype6-dev \
+  libicu-dev \
+  libjpeg-dev \
+  libmcrypt-dev \
+  libmemcached-dev \
+  libpng12-dev \
+  libpq-dev \
+  libxml2-dev \
+  && rm -rf /var/lib/apt/lists/*
+
+#gpg key from https://owncloud.org/owncloud.asc
+RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys E3036906AD9F30807351FAC32D5D5E97F6978A26
+
+# https://doc.owncloud.org/server/8.1/admin_manual/installation/source_installation.html#prerequisites
+RUN docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
+ && docker-php-ext-install gd intl mbstring mcrypt mysql opcache pdo_mysql pdo_pgsql pgsql zip
+
+# set recommended PHP.ini settings
+# see https://secure.php.net/manual/en/opcache.installation.php
+RUN { \
+		echo 'opcache.memory_consumption=128'; \
+		echo 'opcache.interned_strings_buffer=8'; \
+		echo 'opcache.max_accelerated_files=4000'; \
+		echo 'opcache.revalidate_freq=60'; \
+		echo 'opcache.fast_shutdown=1'; \
+		echo 'opcache.enable_cli=1'; \
+	} > /usr/local/etc/php/conf.d/opcache-recommended.ini
+
+# PECL extensions
+RUN pecl install APCu-beta redis memcached
+
+ENV OWNCLOUD_VERSION 6.0.9
+VOLUME /var/www/html
+
+RUN curl -fsSL -o owncloud.tar.bz2 \
+      "https://download.owncloud.org/community/owncloud-${OWNCLOUD_VERSION}.tar.bz2" \
+ && curl -fsSL -o owncloud.tar.bz2.asc \
+      "https://download.owncloud.org/community/owncloud-${OWNCLOUD_VERSION}.tar.bz2.asc" \
+ && gpg --verify owncloud.tar.bz2.asc \
+ && tar -xjf owncloud.tar.bz2 -C /usr/src/ \
+ && rm owncloud.tar.bz2 owncloud.tar.bz2.asc
```

Comment:
Fix indent to lines 41-46 to match the corresponding apache Dockerfile. Should be the same in all four Dockerfiles.